tools/profile.py

```diff
@@ -81,9 +88,9 @@ def stack_id_err(stack_id):
     formatter_class=argparse.RawDescriptionHelpFormatter,
     epilog=examples)
 thread_group = parser.add_mutually_exclusive_group()
-thread_group.add_argument("-p", "--pid", type=positive_int,
+thread_group.add_argument("-p", "--pid", type=positive_int_list,
     help="profile process with this PID only")
-thread_group.add_argument("-L", "--tid", type=positive_int,
+thread_group.add_argument("-L", "--tid", type=positive_int_list,
     help="profile thread with this TID only")
```

Comment:
change this to 'prrofile thread with one or more comman separated TIDs only'.

Reply:
@yonghong-song Thanks for your review, pls check the modification.